src/browser/convert_options.js

```diff
@@ -239,6 +240,12 @@ module.exports = {
             plugins: newOptions.plugins
         };
 
+        // v2Api Fallback for incorrect spelling
```

Comment:
Is there a reason to support the misspelling? This feature is going out in `8.56.27.*`, which is still in staging. Why don't we fix it there?

Reply:
I support that 👍